mne/bem.py

```diff
@@ -657,24 +651,16 @@ def _one_step(mu, u):
         return 1.0
 
     # Compose the data for the linear fitting, compute SVD, then residuals
-    _compose_linear_fitting_data(mu, u)
-    u['uu'], u['sing'], u['vv'] = linalg.svd(u['M'])
-    u['resi'][:] = u['y'][:]
-    for p in range(u['nfit'] - 1):
-        dot = np.dot(u['uu'][p], u['y'])
-        for k in range(u['nterms'] - 1):
-            u['resi'][k] = u['resi'][k] - u['uu'][p, k] * dot
```

Comment:
... and here `uu[p, k]`, i.e. backward, even though they are meant to do the same calculation. This was the bug that got us bad fit values before. Now we get ones like MNE-C gets (RV < 0.01)